Druid/DRBaseOffsetMemoryAddress.class.st

```diff
@@ -70,3 +70,9 @@ DRBaseOffsetMemoryAddress >> rtlArgumentExpressions [
 
 	^ { offsetNode . baseNode }
 ]
+
+{ #category : #SCCP }
+DRBaseOffsetMemoryAddress >> sccpLatticeValueFor: aDRSCCP [ 
+
+	^ aDRSCCP bottom "?"
```

Comment:
I think maybe everything should be bottom by default, right? Instead of putting it in each valuable.

Now, I would leave here the bottom, maybe without the `"?"` comment. I don't see the value of constant propagating an address, because most of the time (always?) they are just operands of load and store instructions.

Reply:
Yes, we can push up this. I prefer to not have foldings by default.